x-pack/plugins/security_solution/server/lib/telemetry/types.ts

```diff
@@ -211,26 +210,25 @@ export interface GetEndpointListResponse {
   per_page: number;
   page: number;
   total: number;
-  data: EndpointExceptionListItem[];
+  data: ExceptionListItem[];
 }
 
 // Telemetry List types
 
-export interface EndpointExceptionListItem {
+export interface ExceptionListItem {
   id: string;
   version: string;
   name: string;
   description: string;
   created_at: string;
-  created_by: string;
   updated_at: string;
-  updated_by: string;
   entries: object;
+  os: string;
   os_types: object;
 }
 
 export interface ListTemplate {
-  trusted_application: TrustedApp[];
-  endpoint_exception: EndpointExceptionListItem[];
-  endpoint_event_filter: EndpointExceptionListItem[];
+  trusted_application: TelemetryEvent[];
```

Comment:
Interesting, why are these types changed to be less concrete?

Reply:
Ref this commit: https://github.com/elastic/kibana/pull/112668/commits/be4c21fa562016d94519a1da988f01f37be9834a
The reasoning around this was 2 fold:
   - `TrustedApp` and `EndpointExceptionListItem` (renamed `ExceptionListItem`) were consolidated, so these would have been 3 `ExceptionListItem`. Using a type we don't own has caused the issue we discussed in the backref issue.
   - The reason for the cast to `TelemetryEvent` is to allow for the allow list filtering without overcomplicating the logic or introducing generics (which wouldn't be easy to do / test)

Reply:
Ah ok, that makes sense.